ios/SitumLocationWrapper.m

```diff
@@ -282,6 +282,28 @@ - (SITLocationRequest *) dictToLocationRequest: (NSDictionary *) dict {
             outdoorLocationOptions.enableOutdoorPositions = [[options valueForKey: @"enableOutdoorPositions"] boolValue];
         }
         
+        if ([options valueForKey: @"updateInterval"]) {
+            outdoorLocationOptions.updateInterval = [[options valueForKey: @"updateInterval"] intValue];
+        }
+        
+        if ([options valueForKey: @"computeInterval"]) {
+            outdoorLocationOptions.computeInterval = [[options valueForKey: @"computeInterval"] intValue];
+        }
+        
+        if ([options valueForKey: @"backgroundAccuracy"]) {
+            NSString *accuracy =  [options valueForKey: @"backgroundAccuracy"];
+
```

Comment:
Are these MAXIMUN, HIGH, MEDIUM, LOW values, documented anywhere? How users know which values to use?
Again they should be constants.